### DISCUSSION TITLE
Custom processor in Nussknacker

### DISCUSSION THREAD

**Discussion by arjun-guptha** [TouK/nussknacker#4932] [Q&A]
I'm a newcomer to Nussknacker, and I'm curious about whether it's possible to create or build a custom processor or component to carry out a specific function within Nussknacker.If any document avaiable please share

**Comment by mslabek** [comment]
Hi @arjun-guptha!

Yes, Nussknacker provides an API for creating custom components, which is described in our documentation [here](https://nussknacker.io/documentation/docs/developers_guide/Components/). 
If you want to see working examples of custom components, you can find them in our repository [here](https://github.com/touk/nussknacker-sample-components).

If you have any more questions, or find that these resources are not enough for your needs, feel free to ask.
